parser/src/AST/Json.hs

```diff
@@ -151,6 +151,22 @@ instance ToJSON Expr where
                 , sourceLocation region
                 ]
 
+          AST.Expression.Literal (Chr chr) ->
+              makeObj
+                  [ type_ "ChrLiteral"
+                  , ("module", JSString $ toJSString "Char")
+                  , ("value", JSString $ toJSString [chr])
+                  , sourceLocation region
+                  ]
+
+          AST.Expression.Literal (Str str _) ->
+              makeObj
+                  [ type_ "StrLiteral"
```

Comment:
I'd prefer "StringLiteral", using the Elm type name, since I expect the end-users of this AST to be mostly familiar with Elm.